### DISCUSSION TITLE
Loop Button for now playing

### DISCUSSION THREAD

**Discussion by N0Y0U2114** [ppy/osu#30995] [Ideas] [answered]
Pretty self explanitory, just play the same song over and over again.

As per usual, have a quick figma mockup

![image](https://github.com/user-attachments/assets/ea96e52d-34d1-4870-a410-5f31774bfa8f)

**Comment by peppy** [comment]
Just go into song select.

**Reply by Joehuu** [reply]
That will not work as it loops back to the preview point. Looping can be done after https://github.com/ppy/osu/pull/30343, where you can add only one song to a collection and it'll technically repeat (just like in stable).

**Reply by peppy** [reply, marked as answer]
It's the best you're getting from osu!. Not a music player sorry.

**Reply by cl8n** [reply]
that seems like a random place to draw the line when the rest of this component is literally emulating a music player...
